core/test-set.ts

```diff
@@ -1,44 +1,51 @@
-import { GlobHelper, TestLoader } from "./_core";
+import { GlobHelper, TestLoader, FileRequirer } from "./_core";
 import { ITestFixture } from "./_interfaces";
 
 const path = require("path");
 
 export class TestSet {
 
-  private _testFixtures: Array<ITestFixture> = [];
+   private _testFixtures: Array<ITestFixture> = [];
 
-  public get testFixtures(): Array<ITestFixture> {
-    return this._testFixtures;
-  }
+   public get testFixtures(): Array<ITestFixture> {
+      return this._testFixtures;
+   }
 
-  public constructor(private _testLoader: TestLoader, private _globHelper: GlobHelper) { }
+   public constructor(private _testLoader: TestLoader, private _globHelper: GlobHelper) { }
```

Comment:
Yeah. I thought you preferred doing this:

    private _testLoader: TestLoader;
    public constructor (testLoader: TestLoader) {
        this._testLoader = testLoader;
    }